spec/controllers/admin/account_moderation_notes_controller_spec.rb

```diff
@@ -29,6 +29,7 @@
 
       it 'fails to create a note' do
         expect { subject }.to_not change(AccountModerationNote, :count)
+        expect(assigns(:moderation_notes).to_a).to eq []
         expect(response).to render_template 'admin/accounts/show'
```

Comment:
I'm not sure if this is a good change? The template would fail to render if it's not assigned — what I probably really want to test is that the `.latest` method is used.